pkg/graph/schema.resolvers.go

```diff
@@ -489,25 +489,34 @@ func (r *mutationResolver) CreateAccessibilityRequest(ctx context.Context, input
 		return nil, err
 	}
 
-	intake, err := r.store.FetchSystemIntakeByID(ctx, input.IntakeID)
-	if err != nil {
-		return nil, err
+	if input.IntakeID == nil && (input.CedarSystemID == nil || len(*input.CedarSystemID) == 0) {
+		return nil, errors.New("Either a system intake ID or a CEDAR system ID is required to create a 508 request")
 	}
 
 	newRequest := &models.AccessibilityRequest{
 		EUAUserID: requesterEUAID,
 		Name:      input.Name,
```

Comment:
Do you think it might make more sense to set the requests `Name` property to `applicationName` later in this function? Kinda means we're ignoring the `name` input, but I think that might be fine